README.md

```diff
@@ -88,10 +88,17 @@ New for Dockerfile
 
 to build docker image : ( in the same folder : e.g. DevOps-Course-Starter-Moudule-2)
 
-$ docker build --tag todo-app .
+to build development and production, run the following command 
 
-to run it 
-
-$ docker run -d -p 5000:5000 --env-file .env todo-app
+Develpment :
+```bash
+$ docker build --target development --tag todo-app:dev . 
+$ docker run -d -p 5000:5000 --env-file .env todo-app:dev
+```
 
-and the in borswer : localhost:5000
\ No newline at end of file
+Production : 
+```bash
+$ docker build --target production --tag todo-app:prod .
+$ docker run -d -p 5000:5000 --env-file .env todo-app:dev
```

Comment:
You've accidentally copied the dev command again here - this one should target a different port & use the right image name!

```suggestion
$ docker run -d -p 5000:8000 --env-file .env todo-app:prod
```